tests/load/pipeline/test_import_export_schema.py

```diff
@@ -0,0 +1,114 @@
+    export_schema.tables["person"]["columns"]["name"]["data_type"] == "text"
+
+    # discovered columns are not present in the import schema
+    import_schema = _get_import_schema(name)
+    assert "id" not in import_schema.tables["person"]["columns"]
+    assert "name" not in import_schema.tables["person"]["columns"]
+
+
+def test_provided_columns_exported_to_import() -> None:
+    name = "schema_test" + uniq_id()
+
+    p = dlt.pipeline(
+        pipeline_name=name,
+        destination="duckdb",
+        import_schema_path=IMPORT_SCHEMA_PATH,
+        export_schema_path=EXPORT_SCHEMA_PATH,
+    )
+
+    p.run(EXAMPLE_DATA, table_name="person", columns={"id": {"data_type": "text"}})
+
+    # discovered columns are in export
+    export_schema = _get_export_schema(name)
+    export_schema.tables["person"]["columns"]["id"]["data_type"] == "bigint"
+    export_schema.tables["person"]["columns"]["name"]["data_type"] == "text"
+
+    # discovered columns are not present in the import schema
+    # but provided column is
+    import_schema = _get_import_schema(name)
+    assert "name" not in import_schema.tables["person"]["columns"]
+    import_schema.tables["person"]["columns"]["id"]["data_type"] == "text"
+
+
+def test_import_schema_is_respected() -> None:
+    name = "schema_test" + uniq_id()
+
+    p = dlt.pipeline(
+        pipeline_name=name,
+        destination="duckdb",
+        import_schema_path=IMPORT_SCHEMA_PATH,
+        export_schema_path=EXPORT_SCHEMA_PATH,
+    )
+    p.run(EXAMPLE_DATA, table_name="person")
+    assert p.default_schema.tables["person"]["columns"]["id"]["data_type"] == "bigint"
+
+    # take default schema, modify column type and save it to import folder
+    modified_schema = p.default_schema.clone()
+    modified_schema.tables["person"]["columns"]["id"]["data_type"] = "text"
+    with open(os.path.join(IMPORT_SCHEMA_PATH, name + ".schema.yaml"), "w") as f:
+        f.write(modified_schema.to_pretty_yaml())
+
+    # run load again import schema is not used as a schema exists
+    p.run(EXAMPLE_DATA, table_name="person")
+    assert p.default_schema.tables["person"]["columns"]["id"]["data_type"] == "bigint"
+
+    # import schema is not overwritten
+    assert _get_import_schema(name).tables["person"]["columns"]["id"]["data_type"] == "text"
+
+    # when creating a new schema (e.g. with full refresh), this will work
+    p = dlt.pipeline(
+        pipeline_name=name,
+        destination="duckdb",
+        import_schema_path=IMPORT_SCHEMA_PATH,
+        export_schema_path=EXPORT_SCHEMA_PATH,
+        full_refresh=True,
+    )
+    p.run(EXAMPLE_DATA, table_name="person")
+    assert p.default_schema.tables["person"]["columns"]["id"]["data_type"] == "text"
+
+    # import schema is not overwritten
+    assert _get_import_schema(name).tables["person"]["columns"]["id"]["data_type"] == "text"
+
+    # export now includes the modified column type
```

Comment:
related to the above, if importing the schema, the updated column will be in the export